src/main/java/org/osc/sdk/controller/api/SdnRedirectionApi.java

```diff
@@ -33,211 +33,230 @@
-     * @throws Exception
+     * Deletes a NetworkElement
+     *
+     * @param element  provides context information of the network element, see {@link NetworkElement}
+     * @throws Exception upon failure
      */
     void deleteNetworkElement(NetworkElement element) throws Exception;
 
     /**
-     * Get the children elements of the network element
-     * For example get the protected workload ports of the Port Group
-     * @param element
-     * @return
-     * @throws Exception
+     * Retrieve the list of NetworkElements
+     *
+     * @param element  provides context information of the network element, see {@link NetworkElement}
+     * @return the list of NetworkElement
+     * @throws Exception upon failure
      */
     List<NetworkElement> getNetworkElements(NetworkElement element) throws Exception;
 
     /**
-     * Install network hooks to inspect traffic of given network port by another network port
-     * which is part of an inspection network device.
-     *
-     * @param networkElem
-     *            The port on which behalf traffic is being inspected or a networkElement containing protected ports
-     * @param inspectionPort
-     *            The inspection port belonging to network inspection devices.
-     * @param tag
-     *            A tag made available to inspection port data path. Usually represent the policy by
-     *            which traffic should be inspected.
-     * @param encType
-     *            Data path Encapsulation Method to use to extract tag and direction from packet
+     * Creates an inspection hook
      *
-     * @param order
-     *            The order in which to insert the inspection hook. If other inspection hooks are already in place, all
-     *            existing hooks will be push down from the point of insertion. The order range is zero based where -1
-     *            symbolizes to append as last
+     * @param networkElem  provides context information of the network element, see {@link NetworkElement}
+     * @param inspectionPort  provides context information of the inspection port element, see{@link InspectionPortElement}
+     * @param tag  provides a tag
+     * @param encType  provides the tag encapsulation type, see{@link TagEncapsulationType}
+     * @param order  provides an order
+     * @param failurePolicyType  provides the failure policy type, see{@link FailurePolicyType}
+     * @return the identifier of the inspection hook
+     * @throws NetworkPortNotFoundException when port is not found
+     * @throws Exception upon failure
      */
     String installInspectionHook(NetworkElement networkElem, InspectionPortElement inspectionPort, Long tag,
             TagEncapsulationType encType, Long order, FailurePolicyType failurePolicyType)
                     throws NetworkPortNotFoundException, Exception;
 
     /**
-     * Remove previously installed network hooks placed on behalf of network port.
+     * Deletes an inspection hook
```

Comment:
Deletes the inspection hook corresponding to the provided network element and inspection port